pkgs/tools/typesetting/polylux2pdfpc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, nix-update-script }:
```

Comment:
What's the point of having this file? Shouldn't it be removed?

Reply:
I could've swore I removed it, but doing it now, thanks.